magstock/templates/registration/checkin.html

```diff
@@ -0,0 +1,154 @@
+<script type="text/javascript">
```

Comment:
Like I said, this is just a copied file, which is kludgy but works.  I'll point out the change I made below.